compiler/plugins/target/AMD-AIE/iree-amd-aie/Target/tests/cdo/matmul_16x16_8xi32__dispatch_0_matmul_16x1_0.aiecc.mlir

```diff
@@ -161,32 +161,6 @@ aie.device(npu1_4col) {
     aiex.npu.sync {channel = 0 : i32, column = 0 : i32, column_num = 1 : i32, direction = 0 : i32, row = 0 : i32, row_num = 1 : i32}
     return
   }
-  aie.wire(%shim_mux_0_0 : North, %switchbox_0_0 : South)
```

Comment:
I dont follow why the configuration information is removed?

Reply:
it's not config info - these ops don't do anything (ctrl+f `WireOp` [AMDAIETargetCDODirect.cpp](https://github.com/nod-ai/iree-amd-aie/blob/main/compiler/plugins/target/AMD-AIE/iree-amd-aie/Target/AMDAIETargetCDODirect.cpp)). They are meant to be used in combination with [`--aie-find-flows`](https://github.com/Xilinx/mlir-aie/blob/main/include/aie/Dialect/AIE/Transforms/AIEPasses.td#L155-L164) to round-trip the router (flows -> wires -> flows). But that roundtrip is too low resolution (it didn't catch bugs months ago when I was in mlir-aie) so we don't have it (so we don't need these `aie.wire`s).